python/graphstorm/inference/emb_infer.py

```diff
@@ -63,6 +64,8 @@ def infer(self, data, task_type, save_embed_path, eval_fanout,
             graph partition algorithm.
         save_embed_format : str
             Specify the format of saved embeddings.
+        infer_batch_size: int
+            Specify the inference batch size of embeddings.
```

Comment:
```suggestion
            Specify the inference batch size of for computing input embeddings.
```

Reply:
I don't quite understand word "input embeddings" here. I thought we are computing node embeddings in this function.